sdk/nanotdf_policy.go

```diff
@@ -14,10 +17,10 @@ import (
 type policyType uint8
 
 const (
-	policyTypeRemotePolicy                           policyType = 0
-	policyTypeEmbeddedPolicyPlainText                policyType = 1
-	policyTypeEmbeddedPolicyEncrypted                policyType = 2
-	policyTypeEmbeddedPolicyEncryptedPolicyKeyAccess policyType = 3
+	NanoTDFPolicyModeRemote                   policyType = 0
+	NanoTDFPolicyModePlainText                policyType = 1
+	NanoTDFPolicyModeEncrypted                policyType = 2
+	NanoTDFPolicyModeEncryptedPolicyKeyAccess policyType = 3
```

Comment:
Slightly more idiomatic to use the `iota`. Also, export the type.
```suggestion
	NanoTDFPolicyModeRemote                   PolicyMode = iota
	NanoTDFPolicyModePlainText                PolicyMode
	NanoTDFPolicyModeEncrypted                PolicyMode
	NanoTDFPolicyModeEncryptedPolicyKeyAccess PolicyMode
```
Arguably it should just be PolicyModeX? Do we plan to make this nano-only?

https://go.dev/wiki/Iota

Alternative: keep separate internal names that are explicitly valued (0, 1, 2, 3), and use iota for Public variants to allow separation of concerns between the API (what SDK consumers see, the value should be opaque) and the file format (what is used when reading/writing files, value must be known/specified)


(I guess my only real substantive concern here is exporting the type name, the others are all just kinda suggestions/discussion points.)